files/cloudlinux/pes-events.json

```diff
@@ -183716,6 +183716,13 @@
                         ],
                         "name": "kernel-modules-extra",
                         "repository": "almalinux8-baseos"
+                    },
+                    {
+                        "modulestreams": [
+                            null
+                        ],
+                        "name": "kmod-lve",
+                        "repository": "almalinux8-baseos"
```

Comment:
I don't believe that kernel module is located in the Alma repos, right?
Though, also, IIRC, that "repository" line doesn't have much effect on how Leapp resolves the new list of packages to install.